src/test/java/SpaceTest.java

```diff
@@ -0,0 +1,15 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SpaceTest {
+
+    @Test
+    @DisplayName("All spaces should be removed from String")
+    void noSpace() {
+        String input = Space.noSpace("8 j 8   mBliB8g  imjB8B8  jl  B");
```

Comment:
```suggestion
        String result = Space.noSpace("8 j 8   mBliB8g  imjB8B8  jl  B");
```
bildet dann eine Einheit mit der Variablen `expectedResult`